test/Release/Gateway/VisibleStubReleaseGateway.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Volodymyr
+ * Date: 04/08/2015
+ * Time: 12:17
+ */
+
+namespace Clearbooks\Labs\Release\Gateway;
+
+
+use Clearbooks\Labs\Release\Release;
+
+class VisibleStubReleaseGateway implements ReleaseGateway
```

Comment:
ConfigurableVisibilityReleaseGatewayMock

This could be considered a partial fake.